cfc_app/tests.py

```diff
@@ -35,11 +35,34 @@ def test_health_status_is_up(self):
         self.assertContains(response, '{"status": "UP"}')
 
     def test_health_status_redirects(self):
-        """ Test that '/health' is redirected to '/health/' and returned with RC=301 """
+        """ Test that '/health' is redirected to '/health/' with RC=301 """
 
         response = self.client.get('/health')
-        self.assertEqual(response.status_code, 301)
-        self.assertEqual(response.url, '/health/')
+        self.assertRedirects(response, '/locations/', 301, 200)
```

Comment:
found a redirects assertion, which I think would be better to use for redirects test